build_rules.gradle

```diff
@@ -400,6 +415,17 @@ ext.applyGoNature = {
       tasks.getByPath(project.path + ":resolveTestDependencies").mustRunAfter tasks.getByPath(previous + ":installDependencies")
     }
   }
+
+  idea {
+    module {
+      // The gogradle plugin downloads all dependencies into the source tree here,
+      // which is a path baked into golang
+      excludeDirs += file("${project.path}/vendor")
+
+      // Don't know what is in here, but it isn't our source
```

Comment:
This comment isn't very useful, can you check whats in here.

Reply:
Rephrased